llarp/link/endpoint.cpp

```diff
@@ -0,0 +1,72 @@
+#include "endpoint.hpp"
+#include "link_manager.hpp"
+
+namespace llarp::link
+{
+  std::shared_ptr<link::Connection>
+  Endpoint::get_conn(const RouterContact& rc) const
+  {
+    for (const auto& [rid, conn] : conns)
```

Comment:
this could just be `conns.find(rc.pubkey)` I think

Reply:
nice good point